README.md

```diff
@@ -5,6 +5,8 @@ Collection of [syzkaller](https://github.com/google/syzkaller) bug reproducers f
 Obtained from [syzbot](https://github.com/google/syzkaller/blob/master/docs/syzbot.md)
 using [syz-reprolist](https://github.com/google/syzkaller/blob/64612bfd3c83d7246467cd9731f01a36b6331598/tools/syz-reprolist/reprolist.go).
 
+To collect C reproducers from syz-manager workdir, run [./collect_reprogs.py <path/**/*.cprog>](collect_reprogs.py).
```

Comment:
This part "**/*.cprog" is a bit confusing. Do I need to spell this? Also what is "path"?
I see "**/*.cprog" is appended to the glob in the script, so it should not be passed.
User is supposed to pass just manager workdir, right? If so, I think something like "./collect_reprogs.py /path/to/manager/workdir" would be easier to understand for users (including me).

Reply:
True. Fixed.